tests/model_registry/test_model_registry_creation.py

```diff
@@ -0,0 +1,41 @@
+import shlex
+from ast import literal_eval
+from typing import Self
+from simple_logger.logger import get_logger
+from ocp_resources.model_registry import ModelRegistry
+from pyhelper_utils.shell import run_command
+
+from tests.model_registry.utils import generate_register_model_command
+
+LOGGER = get_logger(name=__name__)
+
+
+class TestModelRegistryCreation:
+    """Tests the creation of a model registry"""
+
+    # TODO: Enable Model Registry in DSC if needed
+
+    def test_model_registry_instance_creation(self: Self, model_registry_instance: ModelRegistry):
+        assert model_registry_instance.exists
```

Comment:
you already have the check in `model_registry_instance` 
`wait_for_condition` is  waiting on `self.exists` so if the resource does not exist the setup will not pass leading to pytest error.
if setup succeeds, the test will always pass.
`wait_for_resource=True` will also make sure that the resource is created.